### PR TITLE
Wait for healthy containers before doing tests

### PR DESCRIPTION
**Pull request by diversemix**
Closes: #776

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -67,6 +68,7 @@ test_integration: setup
 	$(MAKE) create_networks
 	-${DOCKER_COMPOSE_INFRA} up -d
 	${DOCKER_COMPOSE} up -d
+	while [ `docker ps | grep healthy | wc -l` != $(NUM_CONTAINERS) ] ; do sleep 1 ; echo -n .;done
```
Comment (diversemix) [inline Makefile:71]: Do we think we need a timeout on this?

Comment (Cooryd) [reply Makefile:71]: Theres a util in .scripts that does this with a timeout arg if I remember correctly

Comment (diversemix) [reply Makefile:71]: Yeah - thanks - its seems there is a remnant of it in the Makefile ... let me re-do this :)

Comment (diversemix) [reply Makefile]: done

**Pull request merged**